easybuild/easyconfigs/z/zlib/zlib-1.2.8-GCC-4.7.3.eb

```diff
@@ -9,6 +9,7 @@ not covered by any patents -- lossless data-compression library for use on virtu
 computer hardware and operating system."""
 
 toolchain = {'name': 'GCC', 'version': '4.7.3'}
+toolchainopts = {'pic': True}
```

Comment:
Yes, indeed, I noticed that the easyconfig I copied didn't have `pic` enabled, so I tracked down all the ones that didn't have it yet.

Another example why we should step up the effort for fat (`.yeb`) easyconfigs...